de.json

```
@@ -81,9 +81,9 @@
   "destination_arrival_soc_description": "Der erlaubte Mindest-SoC bei Ankunft am Ziel",
   "charger_arrival_soc": "SoC bei Ankunft an einem Ladepunkt",
   "charger_arrival_soc_description": "Der erlaubte Mindest-SoC bei Ankunft an einem beliebigen Weg- oder Ladepunkt",
-  "battery_degradation": "Batteriedegradation",
-  "battery_degradation_description": "Vermutete Batteriedegradation des Fahrzeugs im Vergleich zu einer fabrikneuen Batterie. Standard ist 5% für alle Fahrzeuge",
-  "battery_capacity_description": "Live-Daten der Batteriekapazität, die direkt vom Fahrzeug gemessen wird. Aus den Live-Daten entfernen, um zur Degradation zurückzukehren.",
+  "battery_degradation": "Batterieabnutzung",
```

```suggestion
  "battery_degradation": "Batteriedegradation",
```
Ich würde es so lassen. Es ist kein Ausdruck, den man vorliest, also keine gesprochene Sprache. Wenn man es liest weiß man sofort was gemeint ist.